### PR TITLE
Refactor ReviewCreate.js to also work for update (WIP)

### PR DESCRIPTION
**Pull request by asooge**
Use ReviewCreate for two purposes:
- create reviews
- update reviews `if (props.currentReview)`

Ticket:
[ROAM-8](https://workfromroam.atlassian.net/browse/ROAM-8): Use one component for create and update

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -168,22 +166,12 @@ class App extends React.Component {
...
-        <Route user={user} path='/create-workspace' render={() => (
-          <WorkSpaceCreate
-            user={user}
-            placeId={this.state.placeId}
-            placeData={this.state.placeData}
-            workspaceLocation={this.state.poiLocation}
-            setApp={this.setState.bind(this)}
-          />
-          )} />
-
```
Comment (asooge) [inline src/components/App/App.js:180]: This code is outdated. No longer using this route or `WorkSpaceCreate` component

File: src/components/WorkSpace/WorkSpaceButton.js [hunk 2]
```diff
@@ -1,39 +0,0 @@
...
-                <Link to='/create-workspace'>
-                    <button>Create A WorkSpace</button>
-                </Link>
-            </HashRouter>
-
-        )
-    }
-}
-
-export default WorkSpaceButton
```
Comment (asooge) [inline src/components/WorkSpace/WorkSpaceButton.js:39]: This code is outdated. No longer using `WorkSpaceButton` component

File: src/components/WorkSpace/WorkSpaceCreate.js [hunk 3]
```diff
@@ -1,117 +0,0 @@
...
-          {this.props.placeData ? <Button type="submit" onClick={this.handleSubmit}> Create New WorkSpace </Button> : null}
-      </div>
-      )
-    }
-  }
-
-
-
-
-  export default withRouter(WorkSpaceCreate);
```
Comment (asooge) [inline src/components/WorkSpace/WorkSpaceCreate.js:117]: This code is outdated. No longer using `WorkSpaceCreate`

File: src/components/Review/ReviewCreate.js [hunk 4]
```diff
@@ -14,41 +14,66 @@ import Button from 'react-bootstrap/Button'
...
+        food: defaultValues.food || false,
+        outlet: defaultValues.outlet || false,
+        seating: defaultValues.seating || 3,
+        clean: defaultValues.clean || 3,
         display: 'block',
         redirect: false,
         isChecked: props.isChecked || false,
         submitError: false,
       }
-      this.toggleChange = this.toggleChange.bind(this);
```
Comment (asooge) [inline src/components/Review/ReviewCreate.js:40]: no need to call `.bind` because `toggleChange` is using arrow syntax.

**Review by GMorse19 — APPROVED** [review]
Looks great!

**Pull request merged**